draft-ietf-httpbis-http2bis.xml

```diff
@@ -2526,7 +2526,7 @@ WINDOW_UPDATE Frame {
             Prior to receiving a <xref target="SETTINGS" format="none">SETTINGS</xref> frame that sets a value for
             <xref target="SETTINGS_INITIAL_WINDOW_SIZE" format="none">SETTINGS_INITIAL_WINDOW_SIZE</xref>, an endpoint can only use the default
             initial window size when sending flow-controlled frames.  Similarly, the connection flow-control
-            window is set to the default initial window size until a WINDOW_UPDATE frame is
+            window is set based on the default initial window size until a WINDOW_UPDATE frame is
```

Comment:
"is set to" could be misread as saying that it remains set to that fixed value even as data is sent/received, which would disable flow control until the first WINDOW_UPDATE.  That's clearly a wrong reading, but maybe this wording tweak helps.